fleetspeak/src/server/db/store.go

```diff
@@ -228,9 +228,9 @@ type ClientStore interface {
 	// Writes resource-usage data received from a client to the data-store.
 	RecordResourceUsageData(ctx context.Context, id common.ClientID, rud mpb.ResourceUsageData) error
 
-	// Fetches at most 'limit' resource-usage records for a given client from the data-store.
+	// Fetches resource-usage records for a given client and time range from the data-store.
 	// TODO: Add more complex queries.
```

Comment:
IIUC, the TODO comment can now be removed?

Reply:
I guess it depends on whether we'd like to extend the function to other possible use cases sometime in the future. If not, then we can essentially remove the comment.

Reply:
We might want to extend it at some point, but given that we don't have any well formulated plan on how we'd like to do it, it's safe to remove the comment.